src/analysis.zig

```diff
@@ -51,7 +51,7 @@ pub fn collectDocComments(
         switch (tree.token_ids[curr_line_tok]) {
             .LineComment => continue,
             .DocComment, .ContainerDocComment => {
-                try lines.append(std.fmt.trim(tree.tokenSlice(curr_line_tok)[3..]));
+                try lines.append(std.mem.trim(u8, tree.tokenSlice(curr_line_tok)[3..], &[_]u8{ ' ', '\t', '\n', '\r' }));
```

Comment:
The proper replacement is using `mem.trim` with `ascii.spaces`:
```suggestion
                try lines.append(std.mem.trim(u8, tree.tokenSlice(curr_line_tok)[3..], &std.ascii.spaces));
```

Reply:
I probably should have looked around a bit more instead of hardcoding the characters like `std.fmt.trim` did.

I've update this pull request to use `ascii.spaces`

Reply:
well depending on when you looked it may not have been there yet, I just added it: https://github.com/ziglang/zig/pull/6931